packages/neuron-ui/src/utils/formatters.ts

```diff
@@ -244,16 +243,17 @@ export const sudtValueToAmount = (value: string = '0', decimal: string = '0', sh
     sign = '+'
   }
   const unsignedValue = value.replace(/^-?0*/, '')
+  const dec = +decimal
   if (dec === 0) {
     return +unsignedValue ? `${sign}${unsignedValue}` : '0'
   }
-  let unsignedCKB = ''
+  let unsignedSUDTValue = ''
   if (unsignedValue.length <= dec) {
-    unsignedCKB = `0.${unsignedValue.padStart(dec, '0')}`.replace(/\.?0+$/, '')
+    unsignedSUDTValue = `0.${unsignedValue.padStart(dec, '0')}`.replace(/\.?0+$/, '')
```

Comment:
If this regex pattern is used in more than one place, I would suggest creating a function to encapsulate it.

Reply:
OK, I'll do it later because the formatters of CKB value and sUDT value are going to be merged into a general one when the Send Components of CKB and sUDT are merged.

We separate them deliberately when sUDT is experimental.